debezium-testing/debezium-testing-system/src/test/java/io/debezium/testing/system/tests/mysql/MySqlTests.java

```diff
@@ -37,22 +38,36 @@ public MySqlTests(
         super(kafkaController, connectController, connectorConfig, assertions);
     }
 
-    public void insertCustomer(
-                               SqlDatabaseController dbController,
-                               String firstName, String lastName,
-                               String email)
+    public static void insertCustomer(
```

Comment:
I don't see any reason for static context here. Can you explain?

Reply:
This method is used also in the OcpMysqlReplicationIT, I also considered moving this method to another helper class, but I thought it's unnecessary .